adapter/go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/sirupsen/logrus v1.7.0
 	github.com/streadway/amqp v1.0.0
 	github.com/stretchr/testify v1.7.0
+	github.com/vektah/gqlparser/v2 v2.4.6
```

Comment:
Shall we update it to the latest version?

Reply:
done